### ISSUE TITLE
Full Unicode support

### ISSUE THREAD

**Issue by stuarteberg** [ilastik/ilastik#71]

**Comment by akreshuk** [comment]
Works for label names, dataset nicknames and export file names, on Linux. Anything else, @stuarteberg ?

**Comment by akreshuk** [comment]
Not so fast. Right click on a label name with unicode characters: 
```
Starting ilastik from "/home/akreshuk/miniconda2/envs/ilastik-devel-new".
ERROR 2017-05-23 13:08:50,752 excepthooks 17143 140083432060672 Unhandled exception in thread: 'MainThread'
ERROR 2017-05-23 13:08:50,755 excepthooks 17143 140083432060672 Traceback (most recent call last):
  File "/home/akreshuk/miniconda2/envs/ilastik-devel-new/ilastik-meta/ilastik/ilastik/widgets/labelListView.py", line 95, in contextMenuEvent
    menu.addAction("Clear {}".format(from_name), partial(self.clearRequested.emit, from_index.row(), str(from_name)))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-2: ordinal not in range(128)
```

**Comment by k-dominik** [comment]
Maybe we can consider this issue fixed, with the python 3 switch, after we resolve https://github.com/ilastik/ilastik/issues/1663

**Comment by k-dominik** [comment]
closed via python3 switch, #1663
